moto/ec2/models.py

```diff
@@ -6232,8 +6244,63 @@ def create_transit_gateway_vpc_attachment(
             subnet_ids=subnet_ids,
             options=options
         )
+        self.transit_gateways_attachments[transit_gateway_vpc_attachment.id] = transit_gateway_vpc_attachment
         return transit_gateway_vpc_attachment
 
+    def describe_transit_gateway_attachments(self, transit_gateways_attachment_ids=None, filters=None, max_results=0):
```

Comment:
nit: There's quite some duplication between `describe_transit_gateway_attachments(..)/describe_transit_gateway_vpc_attachments(..)` as well as `describe_transit_gateway_vpc_attachments(..)/describe_transit_gateway_attachments(..)`. Not critical for now, but perhaps in a future iteration we could think about extracting out some of the common functionality...